### PR TITLE
Add diff tests for infinity cases

### PR DESCRIPTION
**Pull request by rumyantseva**
See also https://github.com/FerretDB/FerretDB/pull/1622.

### CODE REVIEW

File: tests/diff/float_values_test.go [hunk 1]
```diff
@@ -29,50 +29,45 @@ func TestFloatValues(t *testing.T) {
 	t.Parallel()
 
 	ctx, db := common.Setup(t)
-
-	t.Run("Insert", func(t *testing.T) {
```
Comment (rumyantseva) [inline tests/diff/float_values_test.go:33]: This subtest is not needed as we only test inserts anyway.

Comment (AlekSi) [reply tests/diff/float_values_test.go:33]: Well, we should also do validation on updates, findAndModify, etc. Let's keep and add a stub for updates

File: tests/diff.yml [hunk 2]
```diff
@@ -5,22 +5,34 @@ args: ["-timeout=30s", "-shuffle=on", "-race", "./..."]
...
     pass:
       - regex: FerretDB$
 
     fail:
       - not_regex: FerretDB$
 
+    # https://github.com/FerretDB/dance/issues/266
+    skip:
+      - github.com/FerretDB/dance/tests/diff/TestFloatValues/FindAndModify
+      - github.com/FerretDB/dance/tests/diff/TestFloatValues/Update
```
Comment (rumyantseva) [inline tests/diff.yml:21]: @AlekSi this configuration doesn't seem to be correct (as dance fails), but I don't understand why. What did I do wrong?

**Review by w84thesun — DISMISSED** [review]

**Review by w84thesun — APPROVED** [review]

**Review by chilagrow — APPROVED** [review]

**Review by AlekSi — COMMENTED** [review]
LGTM

**Pull request merged**